app/database/sql.py

```diff
@@ -1,7 +1,8 @@
 import databases
-
+import logging
 from ..utils.singletonMeta import SingletonMeta
 
+logger = logging.getLogger(__name__)
```

Comment:
you have used 2 logger objects.  Later we will have requirement to use in many API files.
Is it to have a singleton class for logger ( like we have for database) ?

Reply:
No. It is better to declare loggers for each module this way. Later in time if we have to log messages based on ancestral logger propagation, it will be helpful. Also, this is the recommended way as it helps us in identifying the origin of log message.